beacon-chain/powchain/testing/BUILD.bazel

```diff
@@ -4,8 +4,8 @@ go_library(
     name = "go_default_library",
     testonly = True,
     srcs = [
-        "faulty_mock.go",
-        "mock.go",
+        "faulty_mock_powchain.go",
```

Comment:
`mock_faulty_powchain` is more obvious for its intend. I prefer to always prefix with mock